neuralmonkey/nn/bidirectional_rnn_layer.py

```diff
@@ -43,34 +43,33 @@ def outputs_bidi(self):
 
     @property
     def encoded(self):
-        """States of the bidirectional layer"""
+        """Last state of the bidirectional layer"""
         return tf.concat(1, [self._last_state, self._last_state_rev])
 
 
-    def _reverse_seq(self, input_seq, lengths):
-        """Reverse a list of Tensors up to specified lengths.
+def _reverse_seq(input_seq, lengths):
```

Comment:
To už můžeme zahodit a zavolat z TensorFlow (https://www.tensorflow.org/versions/r0.10/api_docs/python/array_ops.html#reverse_sequence). Zbavíme se tak zakázaného import array_ops.